Tests/PackageLoading/ConventionTests.swift

```diff
@@ -440,6 +440,95 @@ class ConventionTests: XCTestCase {
         }
     }
 
+    func testValidSources() throws {
+        var fs = InMemoryFileSystem()
+        try fs.createEmptyFiles("/main.swift",
```

Comment:
Completely off topic, but I wonder whether we should really support this particular convention... it seems pretty rarely used, and it is confusing to have the Swift files adjacent to the manifest.